CHANGELOG.md

```diff
@@ -1,3 +1,271 @@
+# 23.03 (2022-03-31)
+
+> Checkpoint: `5323020c3f02b178f2b6807f13d89bf7f40d3cce`
+
+For the full log, check out the [git log](https://github.com/helix-editor/helix/compare/22.12..23.03).
+
+Breaking changes:
+
+- Select diagnostic range in `goto_*_diag` commands (#4713, #5164, #6193)
+- Remove jump behavior from `increment`/`decrement` (#4123, #5929)
+- Enable the Kitty keyboard protocol on terminals with support (#4939, #6170, #6194)
+- Select change range in `goto_*_change` commands (#5206)
+- Split file modification indicator from filename statusline elements (#4731, #6036)
+- Jump to symbol ranges in LSP goto commands (#5986)
+
+Features:
+
+- Dynamic workspace symbol picker (#5055)
+- Soft-wrap (#5420, #5786, #5893, #6440)
+- Initial support for LSP snippet completions (#5864, b1f7528, #6263, bbf4800, 90348b8, f87299f, #6371)
+- Add a statusline element for showing the current version control HEAD (#5682)
+- Display LSP type hints (#5934, #6312)
```

Comment:
I don't want to claim the credit here but I think mentioning #5420 would make sense here since that PR added all the virtual text API/implementation and is therefore the main reason we have this feature at all. We usually don't mention PRs twice I believe but we could make an exception for that PR since it was so large